src/frontend/components/administration/apikey/CreateDialog.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <v-dialog :value="visible" max-width="1000px" @input="closeDialog">
```

Comment:
`@input` is invoked whenever the model changes in the VDialog component, and is rather used to update `visible` with the new value rather than setting it to `false`

Reply:
`visible` is a prop so you can't edit it directly. [`closeDialog` is just a `this.$emit('closed')`](https://github.com/ezpaarse-project/ezunpaywall/blob/5a71a60ce30e8ff8690f3969bf2227da0d893883/src/frontend/components/administration/apikey/CreateDialog.vue#L120). The `closed` event [is listened by `ApikeyTab`](https://github.com/ezpaarse-project/ezunpaywall/blob/5a71a60ce30e8ff8690f3969bf2227da0d893883/src/frontend/components/administration/apikey/ApikeyTab.vue#L18) who [sets the `visible` prop to `false`](https://github.com/ezpaarse-project/ezunpaywall/blob/5a71a60ce30e8ff8690f3969bf2227da0d893883/src/frontend/components/administration/apikey/ApikeyTab.vue#L75).

So it's fine to be it like this.

However, it'll be better to rename `visible` to `value` and refactor `this.$emit('closed')` to `this.$emit('input', false)` in order to use `v-model` (or `this.$emit('closed')` to `this.$emit('update:visible', false)` in order to use `:visible.sync`)